aptos-move/aptos-vm/src/sharded_block_executor/sharded_executor_service.rs

```diff
@@ -214,20 +236,29 @@ impl<S: StateView + Sync + Send + 'static> ShardedExecutorService<S> {
                         self.shard_id,
                         transactions.num_txns()
                     );
+                    let execute_block_timer =
+                        APTOS_SHARDED_EXECUTOR_EXECUTE_BLOCK_SECONDS.start_timer();
                     let ret = self.execute_block(
                         transactions,
                         state_view.as_ref(),
                         concurrency_level_per_shard,
                         maybe_block_gas_limit,
                     );
                     drop(state_view);
+                    execute_block_timer.stop_and_record();
+                    let _timer = APTOS_SHARDED_EXECUTOR_RESULT_TX_SECONDS.start_timer();
                     self.coordinator_client.send_execution_result(ret);
                 },
                 ExecutorShardCommand::Stop => {
                     break;
                 },
             }
         }
+        info!(
```

Comment:
These info logs are going to be spammy and also not useful given we already have the counters?

Reply:
Removed